{{cookiecutter.package_name}}/environment.sh

```diff
@@ -77,6 +77,7 @@ else
   echo "Existing environment found for $env_name."
   one_week_ago=$(date -d "7 days ago" '+%Y-%m-%d %H:%M:%S')
   creation_time="$(head -n1 $CONDA_PREFIX/conda-meta/history)"
+  creation_time=$(echo $creation_time | sed -e 's/^==>\ //g' -e 's/\ <==//g')
```

Comment:
Can you explain what's goiung on here?

Reply:
This is formatting the creation time we get from the the meta history file. It removes the "==>" and "<==" that surround the time.

Reply:
Was this a bug that you are fixing? I don't see how it's related to condaforge.

Reply:
In a way yes, the times were in different formats and it was comparing them correctly but I am more confident now that they are all in the same format.